src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -385,3 +385,38 @@ g.test('shader_module,device_mismatch')
   )
   .paramsSubcasesOnly(u => u.combine('isAsync', [true, false]).combine('mismatched', [true, false]))
   .unimplemented();
+
+g.test('vertex_buffer_layout_must_be_valid')
```

Comment:
Many of these are covered by `vertex_state.spec.ts`, please check that file and try to deduplicate anything (and if there are cases covered here that should be there, add TODOs there).

(fyi I found vertex_state.spec.ts by simply searching all files for `arrayStride` and looking at files that might have covered this test)

Reply:
Done, thanks!